packages/components/HOW-TO-TEST-A-COMPONENT-IN-CLOUD-UI.md

```diff
@@ -74,4 +74,12 @@ You can install the "remote" package directly via `yarn` using the following com
 yarn add '@hashicorp/design-system-components@https://github.com/hashicorp/design-system#head=BRANCH-NAME&workspace=@hashicorp/design-system-components'
 ```
 
+Now, for how things work and are wired up in Cloud UI, this should be done for every engine (there are multiple `package.json` files declaring `@hashicorp/design-system-components` as dependency). So you have to find a way to replace them all at once. One option is to follow these steps:
+
+- run the command above in the roof of the repo; this will add a new entry in the `dependency` block of the `/package.json` file (you will remove it later)
```

Comment:
```suggestion
- run the command above in the root of the repo; this will add a new entry in the `dependency` block of the `/package.json` file (you will remove it later)
```

small typo